modes/telega/evil-collection-telega.el

```diff
@@ -0,0 +1,206 @@
+;;; evil-collection-telega.el --- Evil bindings for calc -*- lexical-binding: t -*-
+
+;; Copyright (C) 2018 Pierre Neidhardt
+
+;; Author: Ruoyu Feng <mail@vonfry.name>
+;; Maintainer: James Nguyen <james@jojojames.com>, Pierre Neidhardt <mail@ambrevar.xyz>
+;; URL: https://github.com/emacs-evil/evil-collection
+;; Version: 0.0.6
+;; Package-Requires: ((emacs "25.1"))
+;; Keywords: evil, emacs, tools, telegram, telega
+
+;; This file is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published
+;; by the Free Software Foundation; either version 3, or (at your
+;; option) any later version.
+;;
+;; This file is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+;;
+;; For a full copy of the GNU General Public License
+;; see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+;; Evil bindings for calc.
+
+;;; Code:
+(require 'telega nil t)
+(require 'evil-collection)
+
+(defconst evil-collection-telega-maps '(telega-root-mode-map
+                                        telega-chat-mode-map
+                                        telega-image-mode-map
+                                        telega-webpage-mode-map
+                                        telega-user-button-map
+                                        telega-msg-button-map
+                                        telega-chat-button-map
+                                        telega-sticker-button-map))
+
+(dolist (map evil-collection-telega-maps)
+  (eval `(defvar ,map)))
```

Comment:
can it be omitted?

Reply:
Indeed, it is not necessary. However, many other modes in evil-collection have `defvar` with their maps, so I coded this. They are removed now.